WultraMobileTokenSDK/Operations/Service/WMTOperationsImpl.swift

```diff
@@ -85,10 +100,45 @@ class WMTOperationsImpl: WMTOperations {
     /// Methods of the delegate are always called on the main thread.
     weak var delegate: WMTOperationsDelegate?
     
-    init(powerAuth: PowerAuthSDK, config: WMTConfig) {
+    init(powerAuth: PowerAuthSDK, config: WMTConfig, pollingOptions: WMTOperationsPollingOptions = []) {
         self.powerAuth = powerAuth
         self.networking = WMTNetworkingService(powerAuth: powerAuth, config: config, serviceName: "WMTOperations")
         self.config = config
+        self.pollingOptions = pollingOptions
+        
+        #if os(iOS)
+        if pollingOptions.contains(.pauseWhenOnBackground) {
+            notificationObservers.append(NotificationCenter.default.addObserver(forName: UIApplication.willResignActiveNotification, object: nil, queue: nil) { [weak self] _ in
+                guard let `self` = self else {
+                    return
+                }
+                self.pollingLock.synchronized {
+                    if self.isPollingOperationsInternal {
+                        self.pollingTimer?.invalidate()
+                    }
+                }
+            })
+            notificationObservers.append(NotificationCenter.default.addObserver(forName: UIApplication.didBecomeActiveNotification, object: nil, queue: nil) { [weak self] _ in
+                guard let `self` = self else {
+                    return
+                }
+                self.pollingLock.synchronized {
+                    if self.isPollingPaused {
+                        guard let timer = self.pollingTimer else {
+                            D.error("This is a logical error, timer shouldn't be deallocated when paused")
+                            return
+                        }
+                        self.pollingTimer = nil
+                        self.startPollingOperationsInternal(interval: timer.timeInterval, delayStart: false)
+                    }
+                }
+            })
+        }
+        #endif
```

Comment:
This is OK for now, but once we'll add support for SPM, you'll need a some protocol to abstract this platform specific implementation.